pagopa/io-app
# feat: [PE-614] CGN landing page new copy

## Short description
New copy for CGN landing page

## List of changes proposed in this pull request
- changed translation files
- add new button

## How to test
[figma](https://www.figma.com/design/yOoBGDr1MJpa0LY5lDUFSd/IOapp_CartaGiovaniNazionale?node-id=6102-36458&t=1CciY9sMGswSE3dR-4)
since this change requires modifying remote data the following are needed to test:
- PE-634 depends on https://github.com/pagopa/io-services-metadata/pull/808
- PE-633 depends on https://github.com/pagopa/io-dev-api-server/pull/399

- login into the app
- go to wallet section
- press add to wallet button
- press Carta Giovani Nazionale
- check the new copy here
- press Aggiungi Carta Giovani Nazionale
- check copy on loading screen
  - there is a new option in io-dev-api-server named  "hangOnActivation"
- check copy on success screen
